src/python/artm/library.py

```diff
@@ -487,7 +502,11 @@ def Overwrite(self, topic_model, commit = True):
 
     if commit:
       timeout = -1
-      HandleErrorCode(self.lib_, self.lib_.ArtmWaitIdle(self.master_id_, timeout))
+      args = messages_pb2.WaitIdleArgs()
+      args.timeout_milliseconds = timeout
+      args_blob = args.SerializeToString()
+      args_blob_p = ctypes.create_string_buffer(args_blob)
+      HandleErrorCode(self.lib_, self.lib_.ArtmWaitIdle(self.master_id_, len(args_blob), args_blob_p))
```

Comment:
[Minor comment] Here it might be nice to reuse WaitIdle() method from the python interface -- without arguments. I should have done it earlier (when I've added the commit operation) -- in this case it would be nothing for you to do when ArtmWaitIdle changed its signature.